packages/tslint-config/tslint.json

```diff
@@ -51,6 +51,7 @@
     "no-string-throw": true,
     "no-submodule-imports": false,
     "no-unnecessary-type-assertion": true,
+    "no-unused-variable": [true, {"ignore-pattern": "^_\\d*"}],
```

Comment:
What pattern are we ignoring here?

Reply:
_, _1, _2, etc. There are some places in code where I needed two dummy vars